portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -1915,14 +2013,6 @@ definition {
 				userEmailAddress = "test@www.baker.com");
 		}
 
-		task ("Setup sp samlsp at able.com:8080") {
```

Comment:
Why we remove this step, is it related to these new tests?

Reply:
Great question, good catch!